Userland/Applications/SystemMonitor/main.cpp

```diff
@@ -293,15 +293,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     auto& process_table_view = *process_table_container.find_child_of_type_named<GUI::TreeView>("process_table");
     process_table_view.set_model(TRY(GUI::SortingProxyModel::create(process_model)));
-    for (auto column = 0; column < ProcessModel::Column::__Count; ++column)
-        process_table_view.set_column_visible(column, false);
-    process_table_view.set_column_visible(ProcessModel::Column::PID, true);
-    process_table_view.set_column_visible(ProcessModel::Column::TID, true);
-    process_table_view.set_column_visible(ProcessModel::Column::Name, true);
-    process_table_view.set_column_visible(ProcessModel::Column::CPU, true);
-    process_table_view.set_column_visible(ProcessModel::Column::User, true);
-    process_table_view.set_column_visible(ProcessModel::Column::Virtual, true);
-    process_table_view.set_column_visible(ProcessModel::Column::DirtyPrivate, true);
+
+    for (auto column = 0; column < ProcessModel::Column::__Count; ++column) {
+        process_table_view.set_column_visible(column,
+            Config::read_bool("SystemMonitor"sv, "ProcessesPreferences"sv, ProcessModel::the().column_name(column),
+                ProcessModel::the().is_default_column(column)));
```

Comment:
We already have the `ProcessModel` in a variable, so we don't need to call `ProcessModel::the()` to get it.

```suggestion
        process_table_view.set_column_visible(column,
            Config::read_bool("SystemMonitor"sv, "ProcessesPreferences"sv, process_model.column_name(column),
                process_model.is_default_column(column)));
```